sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/models/DocumentDebugInfo.java

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+package com.azure.search.documents.implementation.models;
```

Comment:
Does this and any of the other newly generated models need to be moved into public API?